### PR TITLE
Removing node-config.yaml references from Nodes bucket

### PR DESCRIPTION
**Pull request by mburke5678**
https://jira.coreos.com/browse/OSDOCS-257

### CODE REVIEW

File: modules/nodes-nodes-garbage-collection-configuring.adoc [hunk 1]
```diff
@@ -5,55 +5,65 @@
...
+  name: set-garbage-collection <1>
+spec:
+  machineConfigPoolSelector:
+    matchLabels: 
+      custom-kubelet: small-pods <2>
+  kubeletConfig: 
+    ImageMinimumGCAge:  <3>
+    ImageGCHighThresholdPercent: <4>
+    ImageGCLowThresholdPercent: <5>
 ----
```
Comment (mburke5678) [inline modules/nodes-nodes-garbage-collection-configuring.adoc]: I guessed on these ^^

File: modules/nodes-nodes-resources-configuring-setting.adoc [hunk 2]
```diff
@@ -72,38 +67,58 @@ $ curl <certificate details> https://<master>/api/v1/nodes/cluster.node22/proxy/
...
+  machineConfigPoolSelector:
+    matchLabels: 
+      custom-kubelet: small-pods <2>
+  kubeletConfig: 
+    EnforceNodeAllocatable:
+      kube-reserved:
+        - "cpu=200m,memory=512Mi"
+      system-reserved:
+        - "cpu=200m,memory=512Mi"  
+----
```
Comment (mburke5678) [inline modules/nodes-nodes-resources-configuring-setting.adoc]: I guessed on these ^^ based on https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go

File: modules/nodes-scheduler-node-affinity-configuring-preferred.adoc [hunk 3]
```diff
@@ -11,17 +11,11 @@ Preferred rules specify that, if the rule is met, the scheduler tries to enforce
 
 The following steps demonstrate a simple configuration that creates a node and a pod that the scheduler tries to place on the node.
 
-. Add a label to a node by editing the node configuration or by executing the `oc label node` command:
+. Add a label to a node using the `oc label node` command:
```
Comment (mburke5678) [inline modules/nodes-scheduler-node-affinity-configuring-preferred.adoc]: Just removed the editing the node config phrase. Any other changes needed?

File: modules/nodes-scheduler-node-affinity-configuring-required.adoc [hunk 4]
```diff
@@ -11,17 +11,12 @@ Required rules *must* be met before a pod can be scheduled on a node.
 
 The following steps demonstrate a simple configuration that creates a node and a pod that the scheduler is required to place on the node.
 
-. Add a label to a node by editing the node configuration or by using the `oc label node` command:
+. Add a label to a node using the `oc label node` command:
```
Comment (mburke5678) [inline modules/nodes-scheduler-node-affinity-configuring-required.adoc]: Just removed the editing the node config phrase. Any other changes needed?

File: modules/nodes-scheduler-node-antiaffinity-configuring.adoc [hunk 5]
```diff
@@ -15,17 +15,12 @@ Required rules *must* be met before a pod can be scheduled on a node.
 
 The following steps demonstrate a simple configuration that creates a node and a pod that the scheduler is required to place on the node.
 
-. Add a label to a node by editing the node configuration or by using the `oc label node` command:
+. Add a label to a node using the `oc label node` command:
```
Comment (mburke5678) [inline modules/nodes-scheduler-node-antiaffinity-configuring.adoc]: Just removed the editing the node config phrase. Any other changes needed?

File: modules/nodes-scheduler-node-antiaffinity-configuring.adoc [hunk 6]
```diff
@@ -60,17 +55,11 @@ Preferred rules specify that, if the rule is met, the scheduler tries to enforce
 
 The following steps demonstrate a simple configuration that creates a node and a pod that the scheduler tries to place on the node.
 
-. Add a label to a node by editing the node configuration or by executing the `oc label node` command:
+. Add a label to a node using the `oc label node` command:
```
Comment (mburke5678) [inline modules/nodes-scheduler-node-antiaffinity-configuring.adoc]: Just removed the editing the node config phrase. Any other changes needed?

**Comment by mburke5678** [comment]
@sjenning I did a find in our text editor for any remaining references to node-config.yaml and attempted to remove and correct the places where it appeared. 
I would appreciate if you or someone on the team can PTAL and see if what I have is right (I guessed in a few places, see comments https://github.com/openshift/openshift-docs/pull/13675/files#r258289263 and https://github.com/openshift/openshift-docs/pull/13675/files#r258289435) and just removed wording in other places. 
Thanks!!

**Comment by mburke5678** [comment]
Jira for followup changes: https://jira.coreos.com/browse/OSDOCS-311

**Comment by mburke5678** [comment]
Also: https://github.com/openshift/openshift-docs/pull/13756

File: modules/nodes-nodes-garbage-collection-configuring.adoc [hunk 7]
```diff
@@ -5,55 +5,65 @@
...
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: set-garbage-collection <1>
+spec:
+  machineConfigPoolSelector:
+    matchLabels: 
+      custom-kubelet: small-pods <2>
+  kubeletConfig: 
+    ImageMinimumGCAge:  <3>
```
Comment (weinliu) [inline modules/nodes-nodes-garbage-collection-configuring.adoc]: is it okay leaving it empty? Got the same question for 

ImageGCHighThresholdPercent: <4>
ImageGCLowThresholdPercent: <5>

Comment (mburke5678) [reply modules/nodes-nodes-garbage-collection-configuring.adoc]: @weinliu I updated this example in another PR:   kubeletConfig:
    ImageMinimumGCAge: 0 <3>
    ImageGCHighThresholdPercent: 85 <4>
    ImageGCLowThresholdPercent: 80 <5>
```
  kubeletConfig:
    ImageMinimumGCAge: 0 <3>
    ImageGCHighThresholdPercent: 85 <4>
    ImageGCLowThresholdPercent: 80 <5>
```

**Pull request merged**